test/remote-config-disabled.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const http = require('http')
+
+const test = require('tape')
+
+test('remote config disabled', function (t) {
+  const server = http.createServer((req, res) => {
+    t.fail('should not poll APM Server for config')
```

Comment:
I would make this check the request URL in case we add some other route in the future which _should_ be made before the timeout.

Reply:
Fixed